.github/workflows/lima_upload_oidc.yml

```
@@ -0,0 +1,54 @@
+name: build and publish images for use with lima-vm
+
+on:
+  push:
+    branches:
+      - lima-upload  
+  schedule:
+    - cron: '0 11 * * 2'
```

```suggestion
on:
  push:
    branches:
      - lima-upload  
  schedule:
    - cron: '0 11 * * 2'
  workflow_dispatch:
```

Add `workflow_dispatch` to allow manual rebuild/deployment of the images